selectrum.el

```diff
@@ -1717,8 +1717,8 @@ defaults to `completion-extra-properties'."
            0 (length displayed-candidate)
            (list
             'mouse-face 'highlight
-            'help-echo
-            "mouse-1: select candidate\nmouse-3: insert candidate"
+            ;; 'help-echo
+            ;; "mouse-1: select candidate\nmouse-3: insert candidate"
```

Comment:
Looking at this, maybe it is enough to remove the "\n" but I have no strong preference with this anyway.

Reply:
I would remove it. I mean this is a horribly annoying help. I always find the Emacs tooltips annoying, but in this case it really makes the UI unusable.